tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/device_info.py

```diff
@@ -42,28 +70,65 @@ def model(self):
     def software(self):
         return self.marionette.find_element(*self._software_locator).text
 
+    def tap_your_rights(self):
+        element = self.marionette.find_element(*self._your_rights_locator)
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._rights_page_locator))
+
+    def tap_your_privacy(self):
+        element = self.marionette.find_element(*self._your_privacy_locator)
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._privacy_page_header_locator))
+
+    def tap_legal_info(self):
+        element = self.marionette.find_element(*self._legal_info_locator)
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._open_source_notices_locator))
+
+    def tap_open_source_notices(self):
+        element = self.marionette.find_element(*self._open_source_notices_locator)
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._notice_text_locator))
+        time.sleep(3)  # loads the HTML file, needs to wait until it completes loading the page
```

Comment:
not sure which mailing list to ask, since I rarely see test related talks.  perhaps I should email davehunt about it first?